lib/fog/rackspace/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fog
   module Rackspace
-    VERSION = "0.1.6"
+    VERSION = "0.2.0"
```

Comment:
If you could leave this out for now I think that would be preferable (I'll change it at release time).

Reply:
reverted this.